ui/lib/pki/addon/components/page/pki-role-details.hbs

```diff
@@ -1,30 +1,35 @@
-<PageHeader as |p|>
-  <p.top>
-    <Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />
-  </p.top>
-  <p.levelLeft>
-    <h1 class="title is-3" data-test-role-details-title>
-      <Icon @name="file-text" @size="24" class="has-text-grey-light" />
-      PKI Role
-      <code>{{@role.name}}</code>
-    </h1>
-  </p.levelLeft>
-</PageHeader>
 <Toolbar>
   <ToolbarActions>
-    <ConfirmAction
-      @buttonClasses="toolbar-link"
-      @onConfirmAction={{this.deleteRole}}
-      @confirmTitle="Delete role?"
-      @confirmButtonText="Delete"
-      data-test-pki-role-delete
-    >
-      Delete
-    </ConfirmAction>
-    <div class="toolbar-separator"></div>
-    <LinkTo class="toolbar-link" @route="overview">Generate Certificate <Icon @name="chevron-right" /></LinkTo>
-    <LinkTo class="toolbar-link" @route="overview">Sign Certificate <Icon @name="chevron-right" /></LinkTo>
-    <LinkTo class="toolbar-link" @route="roles.role.edit" @model={{@role.id}}>Edit <Icon @name="chevron-right" /></LinkTo>
+    {{#if @role.canDelete}}
```

Comment:
I've had some issues with capabilities values being undefined because they haven't loaded yet - do we need to wrap these in a `{{#if @role.updatePath.isLoading}}`? 

Or perhaps I'm just doing something wonky 🤔

Reply:
hm... that sounds very familiar I'll take a look